### PR TITLE
Blocktype shouldn't be translated in zh-CN guide.

### PR DESCRIPTION
**Pull request by abrahamgreyson**

### CODE REVIEW

File: docs/guide-zh-CN/blocktypes.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "Warning:": "警告：",
+    "Note:": "注意：",
+    "Info:": "Info: ",
```
Comment (cebe) [inline docs/guide-zh-CN/blocktypes.json]: no translation for info?

Comment (cebe) [inline docs/guide-zh-CN/blocktypes.json]: also there should be no space after the colon.

Comment (AbrahamGreyson) [inline docs/guide-zh-CN/blocktypes.json]: @cebe Wrong replace :( will change it.

Comment (cebe) [inline docs/guide-zh-CN/blocktypes.json]: if you tell me how it should be translated I can add it myself directly.

Comment (AbrahamGreyson) [inline docs/guide-zh-CN/blocktypes.json]:
```javascript
{
    "Warning:": "警告：",
    "Note:": "注意：",
    "Info:": "补充：",
    "Tip:": "技巧："
}
```

**Comment by cebe** [comment]
Awesome, thank you!

**Pull request closed without merging**